docker/dev/docker-entrypoint.sh

```diff
@@ -239,7 +239,7 @@ if [ "$1" = 'runtests' ]; then
     export DJANGO_SETTINGS_MODULE="${DJANGO_SETTINGS_MODULE}"_test
 
     set -x
-    args="rdrf/rdrf/testing/unit rdrf/report/tests"
+    args="rdrf/rdrf/testing/unit rdrf/report/tests rdrf/useraudit/tests"
```

Comment:
Need to include `rdrf/useraudit/tests` below in runtests_coverage.

Reply:
Fixed

Reply:
I can't see the changes for some reason. 
Also the CI Checks are still not executing the useraudit unit tests:
```
  [ OK ] Mon Jun 19 06:11:58 UTC 2023 - connected to dbtest 5432
+ args='rdrf/rdrf/testing/unit rdrf/report/tests'
+ '[' '' '!=' '' ']'
+ cd /app
+ coverage run -m pytest rdrf/rdrf/testing/unit rdrf/report/tests
```

Reply:
My bad, must've missed that.